pkg/apis/core/v1/resource_genclient.go

```diff
@@ -101,7 +102,17 @@ func (rwt ResourceWithTag) FilterAPIRequest(ctx context.Context, req *http.Reque
 		return nil, err
 	}
 
-	baseURL := strings.TrimSuffix(req.URL.String(), endpointURL.String())
+	op, err := types.OperationFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	reqURL := req.URL
```

Comment:
take FilterRequestURLHook from #121 into a new PR, get it merged and use it (together with FilterRequestBodyHook) instead of ResourceWithTag.FilterAPIRequest

Reply:
new PR: #123